web/src/views/HomepageView.jsx

```diff
@@ -20,17 +20,15 @@ export default function HomepageView() {
         <Carousel autoplay effect="fade" style={contentStyle}>
             <div>
                 <Image 
-                    width={1200}
-                    height={675}
+                    style={contentStyle}
                     src={process.env.PUBLIC_URL + '/carousel-1-image.jpeg'}
                     preview={false}
                 />
             </div>
 
             <div>
                 <Image 
-                    width={1200}
-                    height={675}
+                    style={contentStyle}
```

Comment:
Okay, one last thing 😄 I see that you removed the inline styling, is it possible that you forgot to push the updated CSS file ? I can't see where contentStyle is defined in this commit.

Reply:
Ok, it must have slipped by me. contentStyle is defined in the HomepageView.jsx file